flink-cdc-runtime/src/test/java/org/apache/flink/cdc/runtime/operators/transform/PostTransformOperatorTest.java

```diff
@@ -780,6 +795,76 @@ void testTimestampTransform() throws Exception {
         transformFunctionEventEventOperatorTestHarness.close();
     }
 
+    @Test
+    void testUnixTimestampTransform() throws Exception {
+        PostTransformOperator transform =
+                PostTransformOperator.newBuilder()
+                        .addTransform(
+                                UNIX_TIMESTAMP_TABLEID.identifier(),
+                                "col1, FROM_UNIXTIME(44) as from_unixtime,"
```

Comment:
One suit of test case may not be sufficient. Still suggest something like https://github.com/apache/flink-cdc/pull/3698#discussion_r1899936303